dev/date-picker.html

```diff
@@ -7,9 +7,22 @@
     <title>Date picker</title>
     <script type="module">
       import '@vaadin/vaadin-date-picker';
+      import '@vaadin/horizontal-layout';
+      import '@vaadin/password-field';
+      import '@vaadin/text-area';
```

Comment:
nit: wouldn't add these extra components/layouting to date-picker.html page which I'd assume to only include the date-picker. Maybe there could be separate dev pages for stuff like field alignment in flexbox

Reply:
Sorry, pushed it by mistake, will revert.